sdk/dapp-kit/src/hooks/wallet/useSignAndExecuteTransactionBlock.ts

```diff
@@ -75,7 +79,7 @@ export function useSignAndExecuteTransactionBlock({
 			}
 
 			if (executeFromWallet) {
-				const walletFeature = currentWallet.features['sui:signAndExecuteTransactionBlock'];
+				const walletFeature = currentWallet.features['sui:signAndExecuteTransactionBlock:v2'];
```

Comment:
We should fallback I imagine?

Reply:
fallback is handled by wallet-standard, which we depend on, so no need to duplicate here